src/common/util/copy-clipboard.ts

```diff
@@ -0,0 +1,8 @@
+export const copyToClipboard = (str) => {
+  const el = document.createElement("textarea");
```

Comment:
We now lose all formatting, we could do the same with a div and HTML, but that would be less secure...

Reply:
When would the formatting be used? GitHub and Discord both don't support formatting.

Reply:
Github detects that this is table information and then handles it accordingly, e.g. when creating an issue.

Reply:
I'll be damned, that's a nice feature.

Reply:
So it's not such a nice feature after all - it doesn't work properly.

Pasting this:

```
System Health
Key 	Value
installation_type 	Home Assistant Core
version 	0.118.0.dev0
dev 	true
hassio 	false
virtualenv 	true
python_version 	3.8.5
docker 	false
arch 	x86_64
timezone 	America/Los_Angeles
os_name 	Linux
os_version 	5.4.0-52-generic

Home Assistant Cloud
Key 	Value
logged_in 	true
subscription_expiration 	27 november 2020 01:00
relayer_connected 	true
remote_enabled 	false
remote_connected 	false
alexa_enabled 	false
google_enabled 	false
can_reach_cert_server 	ok
can_reach_cloud_auth 	ok
can_reach_cloud 	ok

Lovelace
Key 	Value
dashboards 	2
mode 	auto-gen
resources 	1
```

Will result in dropping the headers and anything past the first table:


Key | Value
-- | --
installation_type | Home Assistant Core
version | 0.118.0.dev0
dev | true
hassio | false
virtualenv | true
python_version | 3.8.5
docker | false
arch | x86_64
timezone | America/Los_Angeles
os_name | Linux
os_version | 5.4.0-52-generic

Reply:
(fun fact, I had to use the dev tools to add the raw text as GitHub would intercept it each time and reformat)

Reply:
```ts
export const copyTemplateResultToClipboard = (content: TemplateResult) => {
  const el = document.createElement("div");
  render(content, el);
  document.body.appendChild(el);

  const selection = window.getSelection()!;
  selection.removeAllRanges();
  const range = document.createRange();
  range.selectNodeContents(el);
  selection.addRange(range);

  document.execCommand("copy");

  selection.removeAllRanges();
  document.body.removeChild(el);
};
```

Reply:
After discussing this issue with @ludeeus, he suggested to add a GitHub formatting option in a future PR. I like that and suggest we add that in a future PR.